prompt.py

```diff
@@ -757,8 +757,12 @@ def load_smart_contract(self, args):
                     return
 
     def show_mem(self):
-        total = resource.getrusage(resource.RUSAGE_SELF).ru_maxrss
-        totalmb = total / 1000000
+        if sys.platform == 'win32':
+            totalmb = "unknown "  # not supported
+        else:
+            import resource
```

Comment:
keep this import at top of the file please, we try to avoid inline imports.

Reply:
It will not run then :(

Reply:
Oh really? Well, then let's keep it like this ;)